src/StartGg/Provider.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace SocialiteProviders\StartGg;
+
+use GuzzleHttp\RequestOptions;
+use Illuminate\Support\Arr;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+/**
+ * @see https://dev.start.gg/docs/oauth/oauth-overview
+ */
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
```

Comment:
Can you remove this comment?

Reply:
Done. I only added it because several other socialite providers included this comment.